configs/default/SPBE-SPEEDYBEE_F745_AIO.config

```diff
@@ -0,0 +1,105 @@
+# Betaflight / STM32F745 (S745) 4.3.0 Jun 14 2022 / 00:49:48 (229ac66) MSP API: 1.44
+
+board_name SPEEDYBEE_F745_AIO
+manufacturer_id SPBE
+
+# resources
+resource BEEPER 1 D15
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 E09
+resource MOTOR 4 E11
+resource PPM 1 A10
+resource LED_STRIP 1 D12
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 D05
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 D06
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B06
+resource I2C_SDA 1 B07
+resource LED 1 A02
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 4 E02
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 4 E05
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 4 E06
+resource CAMERA_CONTROL 1 B03
+resource ADC_BATT 1 C03
+resource ADC_CURR 1 C02
+resource PINIO 1 C04
+resource FLASH_CS 1 A04
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 E01
+resource GYRO_CS 1 E04
+
+# timer
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer E09 AF1
+# pin E09: TIM1 CH1 (AF1)
+timer E11 AF1
+# pin E11: TIM1 CH2 (AF1)
+timer D12 AF2
+# pin D12: TIM4 CH1 (AF2)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin E09 2
+# pin E09: DMA2 Stream 3 Channel 6
+dma pin E11 1
+# pin E11: DMA2 Stream 2 Channel 6
+dma pin D12 0
+# pin D12: DMA1 Stream 0 Channel 2
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature LED_STRIP
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set current_meter = ADC
```

Comment:
If this AIO includes ESC I was expecting 
```
set motor_pwm_protocol = DSHOT600
```

Reply:
thanks for your review.